wps_shared/wps_shared/fuel_types.py

```diff
@@ -52,22 +54,20 @@ def is_grass_fuel_type(fuel_type: FuelTypeEnum):
     FuelTypeEnum.C6: {"PC": 100, "PDF": 0, "CC": None, "CBH": 7, "CFL": 1.8},
     FuelTypeEnum.C7: {"PC": 100, "PDF": 0, "CC": None, "CBH": 10, "CFL": 0.5},
     FuelTypeEnum.C7B: {"PC": 100, "PDF": 0, "CC": None, "CBH": 10, "CFL": 0.5},
-    # No CBH listed in RB fire intensity class table for D1.
-    # Using default CBH value of 3, as specified in fbp.Rd in cffdrs R package.
-    FuelTypeEnum.D1: {"PC": 0, "PDF": 0, "CC": None, "CBH": 3, "CFL": 1.0},  # TODO: check cfl
+    FuelTypeEnum.D1: {"PC": 0, "PDF": 0, "CC": None, "CBH": 0, "CFL": 0},
     # No CBH listed in RB fire intensity class table for D2.
     # Using default CBH value of 3, as specified in fbp.Rd in cffdrs R package.
-    FuelTypeEnum.D2: {"PC": 0, "PDF": 0, "CC": None, "CBH": 3, "CFL": 1.0},  # TODO: check cfl
+    FuelTypeEnum.D2: {"PC": 0, "PDF": 0, "CC": None, "CBH": 3, "CFL": 1.0},
     FuelTypeEnum.M1: {"PC": 50, "PDF": 0, "CC": None, "CBH": 6, "CFL": 0.8},
     FuelTypeEnum.M2: {"PC": 50, "PDF": 0, "CC": None, "CBH": 6, "CFL": 0.8},
-    'M2_25': {"PC": 25, "PDF": 0, "CC": None, "CBH": 6, "CFL": 0.8, "cffdrs_fuel_type": FuelTypeEnum.M2},
+    "M2_25": {"PC": 25, "PDF": 0, "CC": None, "CBH": 6, "CFL": 0.8, "cffdrs_fuel_type": FuelTypeEnum.M2},
     FuelTypeEnum.M3: {"PC": 0, "PDF": 30, "CC": None, "CBH": 6, "CFL": 0.8},
     FuelTypeEnum.M4: {"PC": 0, "PDF": 30, "CC": None, "CBH": 6, "CFL": 0.8},
     # NOTE! I think having a default CC of 0 is dangerous, I think we should rather just
     # fail to calculate ROS, and say, unknown.
-    FuelTypeEnum.O1A: {"PC": 0, "PDF": 0, "CC": 0, "CBH": 1, "CFL": 1.0},  # TODO: check cfl
-    FuelTypeEnum.O1B: {"PC": 0, "PDF": 0, "CC": 0, "CBH": 1, "CFL": 1.0},  # TODO: check cfl
-    FuelTypeEnum.S1: {"PC": 0, "PDF": 0, "CC": None, "CBH": 1, "CFL": 1.0},  # TODO: check cfl
-    FuelTypeEnum.S2: {"PC": 0, "PDF": 0, "CC": None, "CBH": 1, "CFL": 1.0},  # TODO: check cfl
-    FuelTypeEnum.S3: {"PC": 0, "PDF": 0, "CC": None, "CBH": 1, "CFL": 1.0}  # TODO: check cfl
+    FuelTypeEnum.O1A: {"PC": 0, "PDF": 0, "CC": 0, "CBH": 0, "CFL": 0},
```

Comment:
What's the implication of changing CFL, CBL from 1.0 to 0?

Reply:
Ok, it did have a TODO to check so that's good. These values are used in critical hours for hfi calc, fire calc and asa so we should keep an eye on those.